django_cloud_deploy/workflow/__init__.py

```diff
@@ -416,8 +416,9 @@ def _generate_base_secrets(database_username: str,
         return {'username': database_username, 'password': database_password}
 
     @staticmethod
-    def _load_secret_names(required_service_accounts: Optional[
-            Dict[str, List[Dict[str, Any]]]] = None) -> (List[str], List[str]):
+    def _load_secret_names(
+            required_service_accounts: Dict[str, List[Dict[str, Any]]]
+    ) -> Tuple[List[str], List[str]]:
```

Comment:
Was lint happy with this?

Reply:
Should be, this is the formatting I got from doing `yapf -i`.

Reply:
when i need to lint fix i run yapf -r -i cli cloudlib workflow tests

Reply:
I believe it was optional because of testing.

Reply:
I ran `yapf -r -i cli cloudlib workflow tests` and there was no diff. How do I see why the Kokoro builds failed? When I click on "Details", I get a page not found error.